README.md

```diff
@@ -200,18 +282,49 @@ When there's a need to implicitly dispatch an event with some data for _every_ c
 A common use case for this is to dispatch a `pageview` event for every component in the application that has a `page` property on its `trackingData`:
 
 ```js
-@track({}, { process: (ownTrackingData) => ownTrackingData.page ? {event: 'pageview'} : null })
+@track({}, { process: (ownTrackingData) => ownTrackingData.page ? { event: 'pageview' } : null })
 class App extends Component {...}
 
 ...
 
-@track({page: 'Page1'})
+@track({ page: 'Page1' })
 class Page1 extends Component {...}
 
 @track({})
 class Page2 extends Component {...}
 ```
 
+<details>
+<summary>Example using hooks</summary>
+
+```js
+function App() {
+  const { Track } = useTracking(
+    {},
+    {
+      process: (ownTrackingData) => ownTrackingData.page ? { event: 'pageview' } : null,
+    }
+  );
+
+  return (
+    <Track>
+      <Page1 />
+      <Page2 />
+    </Track>
+  );
+}
+
+function Page1() {
+  useTracking({ page: 'Page1' });
+}
+
+function Page2() {
+  useTracking({});
+}
+```
+
+</details>
```

Comment:
@tizmagik I can't tell whether these collapsible hook versions of the examples are useful. What do you think?

Reply:
I don't hate them haha -- I actually think it might make sense to have the "old" decorator style be the collapsible example, but maybe for some future doc update. This is great for now, thanks!